traffic_ops/app/lib/API/Tenant.pm

```diff
@@ -44,14 +44,16 @@ sub index {
 	my $orderby = $self->param('orderby') || "name";
 	my $rs_data = $self->db->resultset("Tenant")->search( undef, {order_by => 'me.' . $orderby } );
```

Comment:
maybe you can just make this query smarter like

my $rs_data = $self->db->resultset("Tenant")->search( {-in (my-list-of-tenants)}, {order_by => 'me.' . $orderby } );

^^ I can't remember the dbix syntax for that off the top of my head but basically a dbix query that looks like:

select * from tenant where id in (a list of the user's tenant plus it's children)